lupa/lock.pxi

```diff
@@ -12,13 +23,14 @@ cdef class FastRLock:
     wonderful GIL.
     """
     cdef pythread.PyThread_type_lock _real_lock
-    cdef long _owner            # ID of thread owning the lock
+    cdef bint _has_owner        # whether the lock has an owner
```

Comment:
This is not needed. You can see from `_count > 0` that the lock is owned.

Reply:
Fixed in https://github.com/scoder/lupa/pull/165/commits/4c3012a9308bcf5060132ebdce001ed60bf6b2a0